services/alertmanager/alertmanager.go

```diff
@@ -144,9 +132,8 @@ receivers:
 	}
 }
 
-// sendAlerts sends alerts collected in the Registry.
-func (svc *Service) sendAlerts(ctx context.Context) {
-	alerts := svc.r.collect()
+// SendAlerts sends given alerts
```

Comment:
I think we should add something like "it is caller's responsibility to call this method every now and then". We change a logic there, and we need slightly more comments to change our mental model.

Reply:
I left that part out because we have exported the method so I thought that it will be understood that this method will have to be called by a caller outside of this package, seemed a bit redundant to me

will add the comment in the next commit